handbook/handbook.sop.time-off.md

```diff
@@ -0,0 +1,20 @@
+---
+id: r9XHiz16hDCqndyNhwSjB
+title: Time Off
+desc: ''
+updated: 1645004162111
+created: 1645003558624
+---
+
+## Summary
+
+Taking time off from work is important to relax and recharge. This is the quick SOP for taking time off such as holidays and vacations.
+
+## A Few things prior to taking time off
```

Comment:
can you follow the sop format? see [[Sop|dendron://dendron.handbook/templates.sop]]. you can ignore sections that aren't relevant but change this to "Steps"